tests/test_worktree.sh

```
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+#
+# tests/test_worktree.sh — worktree list and cleanup
+#
+
+test_worktree_fails_without_repo() {
+    local sandbox
+    sandbox=$(mktemp -d)
+    cd "$sandbox"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)